keys/signer.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 Coinbase, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package keys
+
+import "github.com/coinbase/rosetta-sdk-go/types"
+
+// Signer is an interface for different curve signers
+type Signer interface {
```

Comment:
Related to https://github.com/coinbase/rosetta-sdk-go/pull/54/files#r449375144 ... let's use pointers for `types.PublicKey` and `types.SigningPayload`